packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income-utils.ts

```diff
@@ -0,0 +1,21 @@
+import { localize } from '@deriv/translations';
+
+export const income_status_codes = {
+    NONE: 'none',
+    PENDING: 'pending',
+    REJECTED: 'rejected',
+    VERIFIED: 'verified',
+    LOCKED: 'locked',
+} as const;
+
+export const poinc_documents_list = [
```

Comment:
@yauheni-deriv, I think you should return this `array` from a `function`. Translations will not reflect if you use `arrays` or `objects` if I remember correctly

```suggestion
export const poinc_documents_list = [
```